types/dispatcher.d.ts

```diff
@@ -93,6 +96,12 @@ declare class Dispatcher extends EventEmitter {
 }
 
 declare namespace Dispatcher {
+  export interface ComposedDispatcher {
+    close: Dispatcher['close'];
+    destroy: Dispatcher['destroy'];
+    dispatch: Dispatcher['dispatch'];
+  }
```

Comment:
this could probably be

```suggestion
  export type ComposedDispatcher = Pick<Dispatcher, 'close' | 'destroy' | 'dispatch'>
```